ibm/mas_devops/roles/suite_app_rollback/README.md

```diff
@@ -0,0 +1,71 @@
+suite_app_rollback
+===============================================================================
+This role is to roll back Maximo Application Suite Applications to an earlier version. Currently, this is designed for Manage Application only. Rollback is possible only in 8.7 and later. From 8.7 onwards, every version comes with a set of supported versions to which the Application can be rolled back. For example, you can roll back Maximo Application Suite from 8.7.x to 8.7.0. 
+This role will rollback the subscription channel for an an installed MAS application after validating:
```

Comment:
I don't think it rolls back the subscription channel, it seems to rollback the installed version of the ManageApp but the subscription stays the same?